content/payments/upi-deeplinks/refunds.mdx

```diff
@@ -361,11 +340,17 @@ Check status of refunds created against a particular `batchID`.
 
 `Created`: This is when the merchant initiates a refund request via APIs or through The Bridge (Setu dashboard). The refund or its processing has not taken place at this point.
 
-`Marked for Refund`: This status indicates that we have received money from the bank for the underlying transaction, but the refund is yet to be reconciled with the settlement calculation for the day.
+`Initiated`: This is when the refund request has been intimated to the banking partner and begins processing the refund. This is the terminal status of a refund, as NPCI does not share information about when a refund hits the customer’s bank account.
 
-`Queued for Refund`: This status indicates that we are able to adjust the refund amount against the settlement amount.
+`Pending`: This status is implied in case the refunds have not been processed due to a low balance or the merchant not transacting for a while. There are 2 reasons to elaborate on this status:
```

Comment:
```suggestion
`Pending`: This status is implied in case the refunds are awaiting to be reconciled or have completed reconciliation, yet not processed due to one of the following reasons:
```